src/snackbar.jsx

```diff
@@ -78,8 +131,14 @@ const Snackbar = React.createClass({
   },
 
   getInitialState() {
+    let open = this.props.open;
+
+    if (open === null) {
+      open = this.props.openOnMount;
+    }
+
     return {
-      open: this.props.openOnMount,
+      open: open,
```

Comment:
I wanted to avoid the edge case where open and openOnMount are both set. I know, that's not commun.
At least this open variable will be removed with `0.15`